src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/x64/multinomial.cpp

```diff
@@ -0,0 +1,134 @@
+// Copyright (C) 2018-2023 Intel Corporation
```

Comment:
Please move it to [common folder](https://github.com/openvinotoolkit/openvino/tree/master/src/plugins/intel_cpu/tests/functional/single_layer_tests/instances/common), due to here is no any x86 specific.

Reply:
Done!